config/routes.rb

```diff
@@ -0,0 +1,71 @@
+Rails.application.routes.draw do
+  # get 'drag_queen/index'
```

Comment:
Here, make sure you set the root to the view that you want, so that when someone views your site they see something at the root index!
```
  root 'drag_queens#index'
```